oracle/blockeligibilityvalidator.go

```diff
@@ -74,16 +74,12 @@ func (v BlockEligibilityValidator) BlockEligible(block *types.BlockHeader) (bool
 	return block.LayerIndex == eligibleLayer, nil
 }
 
-func (v BlockEligibilityValidator) getValidATX(blockEpoch types.EpochId, block *types.BlockHeader) (*types.ActivationTx, error) {
+func (v BlockEligibilityValidator) getRelevantATX(blockEpoch types.EpochId, block *types.BlockHeader) (*types.ActivationTx, error) {
```

Comment:
why rename? what does relevant mean??

Reply:
relevant means "for the relevant epoch" and I didn't want to keep "valid" because we're not verifying validity (it may be contextually invalid, and that's fine).

Reply:
"fine" == "you're still eligible for blocks"

talk to @barakshani if you disagree.

Reply:
Why not GetAssATX?

Reply:
I like @barakshani's suggestion, but I've just refactored this debate away.

There's now a method with a slightly bigger scopre called `getActiveSetSize`. I feel that improves the code in general and also side-steps this naming discussion.